docs/configuration.md

```diff
@@ -0,0 +1,52 @@
+# Configuration
+
+git phlow is configured via `.gitconfig`, in your local workspace, this means that git phlow usese git own internal config and can therefore be placed anywhere git can read it e.g. global .gitconfig in you home folder.
+
+To get started with a a `.gitconfig` in your local repository you must first let git know that the filepath must be included, run this command to do so:
+```sh
+git config --local include.path ../.gitconfig
+```
+
+##### Config Block
+Each configuration is its own block. You can have as many configuration blocks as you want, just as long as they all have the necessary fields. To use a specific block you run a command with the `-t or --target` option with the name of the block. git phlow by default looks for a block named `[phlow]`. So if you name your preferred blovk `phlow` you do not need to use the `-t` flag.
+
+A block consists of following fields.
+
+- `integration-branch` The name of your default integration branch. normally it's master
+- `remote` The name of your remote. By default git sets this to `origin`
+- `service` The service you use for issues.
+- `issue-api` The url to your api where issues are hosted and managed
+- `issue-web` The url to your webpage where issues are hosted and managed
+- `delivery-branch-prefix` The prefix you want your pretested integration system to look for
+
+##### Supported services
+git phlow can be used with **jira** and **github**. Only one service can be used per configuration block.
+To configure git-phlow, set the `service` property to either **jira** or **github**
+- `service = jira`
+- `service = github`
+
+##### Example Configuration with jira
+```ini
+[phlow]
+    remote                 = myfork
+    service                = jira
+    integration-branch     = superintegration
+    issue-api              = https://jira.com
+    issue-web              = https://jira.com
+    delivery-branch-prefix = integrate-me-please
+```
+
+**important**: When entering the url of your issue service providers, it is important that only the base url is provided, which means no `www.mygithubinstance.com:8080/` but `www.mygithubinstance.com:8080`
```

Comment:
If we only support github cloud we should probably not document how to use a non-github.com instance.
Also I'd suggest an issue making us handle this gracefully rather than crashing if we are not getting the exact format we need :)